Automattic/woocommerce-connect-client
# Update/preset packaging readonly

This change causes the add-package modal to disable dimension and weight fields when a preset package is used. A `read_only` field is needed in the box fields to support this. The server needs to support this (PR pending).

Notes: You'll need the corresponding server branch to be able to save. Everything else works.

To test:
- Check this branch out and try switching around the presets in the drop-down when adding a package
- Check that after saving the box, editing results in the correct fields being read only.
- See that everything saves correctly, etc.

Closes #260 
Closes #303 
Closes #307

@allendav @jeffstieler @jkudish
